store/sqlstore/store.go

```diff
@@ -1025,6 +1079,15 @@ func (ss *SqlStore) RemoveIndexIfExists(indexName string, tableName string) bool
 	return true
 }
 
+func IsConstraintAlreadyExistsError(err error) bool {
```

Comment:
Use case statement and make error numbers constants.

Reply:
Done.